MAUI/NumericEntry/Overview.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: About .NET MAUI NumericEntry control | Syncfusion
+description: Learn here all about introduction of Syncfusion .NET MAUI NumericEntry (SfNumericEntry) control, its features, and more.
+platform: MAUI
+control: SfNumericEntry
+documentation: ug
+---
+
+# .NET MAUI NumericEntry Overview
+
+The .NET MAUI NumericEntry control is designed to provide a user-friendly and advanced input experience for numeric values. It supports a wide range of numeric formats, including currency, percentage, and decimals, among others. With its key features, it enables enhanced user experience and facilitates input validation.
```

Comment:
The .NET MAUI NumericEntry control is designed to provide a user-friendly and advanced input experience for numeric values. It supports a wide range of numeric formats, including currency, percentages, decimals, and more. With its key features, it enables enhanced user experience and facilitates input validation.

Reply:
The requested content modification has been successfully addressed